app/workers/notify_vacancy_on_slack_worker.rb

```diff
@@ -0,0 +1,16 @@
+require "slack_notifier"
+
+class NotifyVacancyOnSlackWorker
+  include Sidekiq::Worker
+  include Rails.application.routes.url_helpers
+
+  def perform(vacancy_id)
+    vacancy = Vacancy.find(vacancy_id)
+    message = %Q(
```

Comment:
I like the idea of having a class to generate a message, although it can be a little "overengineering" if you know what I mean. The class is named `NotifyVacancyOnSlackWorker`, I don't see a problem leaving the `message` here.